connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java

```diff
@@ -39,6 +39,18 @@ public class SparkIntegrationTest extends BaseCRUDTest {
 
   private final File dataDir = new File(System.getProperty("java.io.tmpdir"), "spark_test");
 
+  @Test
+  public void testCreateSchema() throws ApiException {
+    createCommonResources();
+    SparkSession session = createSparkSessionWithCatalogs(CATALOG_NAME);
+    session.catalog().setCurrentCatalog(CATALOG_NAME);
```

Comment:
does this work with `spark_catalog`?

Reply:
I extended the test suite to also test with `spark_catalog`